src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/resource/VisitResourceIntTest.kt

```diff
@@ -554,10 +573,41 @@ class VisitResourceIntTest : IntegrationTestBase() {
         .jsonPath("$.visitorSupport.length()").isEqualTo(updateRequest.supportList!!.size)
         .jsonPath("$.visitorSupport[0].supportName").isEqualTo(updateRequest.supportList!![0].supportName)
         .jsonPath("$.visitorSupport[0].supportDetails").isEqualTo(updateRequest.supportList!![0].supportDetails!!)
-        .jsonPath("$.visitorConcerns").isEqualTo(updateRequest.visitorConcerns!!)
+        .jsonPath("$.visitNotes.length()").isEqualTo(updateRequest.visitNotes!!.size)
+        .jsonPath("$.visitNotes[0].type").isEqualTo(updateRequest.visitNotes!![0].type.name)
+        .jsonPath("$.visitNotes[1].type").isEqualTo(updateRequest.visitNotes!![1].type.name)
         .jsonPath("$.sessionId").isEqualTo(updateRequest.sessionId!!)
     }
 
+    @Test
+    fun `put visit by reference - delete notes`() {
+
+      val updateRequest = UpdateVisitRequest(
```

Comment:
This needs to be explicit on which not is being deleted / removed

Reply:
Yep I suppose so,... even though it makes no difference to the type as its generic code... but I suppose that is the dev knowledge and I can't test like that!

Reply:
actually had a second look at it and I think it's okay as it as there is no code that uses what the type it is! will chant about it!

Reply:
This should be dealt with during amend storey VB-600
PUT /visit/{ref}/notes/